tools/gendocs/gendocs/microshift.go

```diff
@@ -0,0 +1,38 @@
+package gendocs
+
+import (
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// a set of commands excluded from microshift
```

Comment:
Are excluding these, or adding just the ones in the Google doc?

Reply:
The logic in gendocs.go on lines 79-81 says that if it sees a command in this list it skips producing output.